helm/prefect-server/templates/hasura/_helpers.tpl

```diff
@@ -0,0 +1,51 @@
+{{- define "hasura.name" -}}
+{{- default "hasura" .Values.hasura.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- define "hasura.fullname" -}}
+{{- printf "%s%s" (include "prefect-server.fullname" . ) "-hasura" -}}
+{{- end -}}
+
+{{- define "hasura.fqdn" -}}
+{{- $name := include "hasura.fullname" . -}}
+{{- $ns := include "global.namespace" . }}
+{{- $suffix := .Values.global.fqdnSuffix }}
+{{- printf "%s.%s.%s" $name $ns $suffix -}}
+{{- end -}}
+
+{{- define "hasura.api-url" -}}
+{{- $host := include "hasura.fqdn" . -}}
+{{- $port := .Values.global.hasura.port | toString -}}
+{{ printf "http://%s:%s/v1alpha/graphql" $host $port }}
```

Comment:
```suggestion
{{ printf "http://%s:%s/v1alpha1/graphql" $host $port }}
```

Reply:
Still wasn't letting my push, nbd I can figure it out later. Here's the change ^